lib/compat/wordpress-6.5/fonts/class-wp-font-collection.php

```diff
@@ -44,6 +44,54 @@ final class WP_Font_Collection {
 		 */
 		private $src;
 
+		/**
+		 * Font collection sanitization schema.
+		 *
+		 * @since 6.5.0
+		 *
+		 * @var array
+		 */
+		const COLLECTION_SANITIZATION_SCHEMA = array(
+			'name'          => 'sanitize_text_field',
+			'description'   => 'sanitize_text_field',
+			'font_families' => array(
+				array(
+					'font_family_settings' => array(
+						'name'       => 'sanitize_text_field',
+						'slug'       => 'sanitize_title',
+						'fontFamily' => 'sanitize_text_field',
+						'preview'    => 'sanitize_url',
+						'fontFace'   => array(
+							array(
+								'fontFamily'            => 'sanitize_text_field',
+								'fontStyle'             => 'sanitize_text_field',
+								'fontWeight'            => 'sanitize_text_field',
+								'src'                   => 'sanitize_url',
```

Comment:
What about when src is an array?

Reply:
Added a custom method to do that: https://github.com/WordPress/gutenberg/pull/58636/commits/216d2106cb56478eab2a3cfbe127bf28ba4d98e3